activerecord/lib/active_record/attribute_methods.rb

```diff
@@ -463,6 +457,16 @@ def attributes_for_create(attribute_names)
         end
       end
 
+      def attribute_value_for_inspect(value)
```

Comment:
I'm not sure about the name of this method... I think something like `format_for_inspect(value)` reveals better the method's intention. Having said this... I also dislike the public API's method name `attribute_for_inspect`, but that isn't in the scope of this PR.